### PR TITLE
Chore: update font

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- Updated the font to Roboto
- Removed variables
- update teh CSSs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.7",
     "redux-saga": "^0.16.0",
+    "twin.macro": "1.2.0",
```
Comment (simboonlong) [inline package.json]:
```suggestion
    "twin.macro": "^1.12.1",
```

get the latest 1-ish version?

Comment (isaackps) [reply package.json]: ok

File: src/components/AssetManagementPanel/AssetInformationPanel/index.tsx [hunk 2]
```diff
@@ -31,13 +30,10 @@ export const AssetInformationPanel: FunctionComponent<AssetInformationPanel> = s
   }
 )`
   .endorsement-chain-button {
-    color: ${vars.brandBlue};
-    cursor: pointer;
-    transition: color 0.3s ${vars.easeOutCubic};
+    ${tw`text-blue cursor-pointer transition duration-300 ease-out`}
```
Comment (simboonlong) [inline src/components/AssetManagementPanel/AssetInformationPanel/index.tsx]: shall we be abit more explicit on transition property? im guessing its only colors so it should be `transition-colors`

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx [hunk 3]
```diff
@@ -1,109 +1,70 @@
...
-
-  @media only screen and (min-width: ${vars.lg}) {
-    min-height: 600px;
-  }
+  ${tw`text-center p-6 flex flex-col justify-center rounded-xl min-h-400 lg:min-h-600`}
 
   &.default {
-    ${mixin.dropzoneStatus({ bg: vars.blueLightest, borderColor: vars.brandBlue })}
+    ${mixin.dropzoneStatus({ bg: "#f3f8fc", borderColor: "#0099cc" })}
   }
```
Comment (simboonlong) [inline src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx:10]: yeah the hardcoded color values should be fine for now, we should think about how to manage again next time

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx [hunk 4]
```diff
@@ -1,109 +1,70 @@
...
+    
     .unverified-btn {
-      ${mixin.button({
-        bg: vars.invalid,
-        bgHover: lighten(0.05, vars.invalid),
-        color: vars.white,
-        borderColor: vars.invalid,
-      })}
+      ${tw`transition duration-200 ease-out border border-solid border-red-400 text-white bg-red-400 py-2 px-6 rounded font-medium text-center align-middle min-w-135 cursor-pointer hover:border-red-300 hover:bg-red-300 hover:shadow-md hover:no-underline`}
     }
```
Comment (simboonlong) [inline src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx]: explicit transition property

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx [hunk 5]
```diff
@@ -1,109 +1,70 @@
...
-  ${mixin.button({
-    bg: vars.white,
-    bgHover: vars.brandBlue,
-    color: vars.brandBlue,
-    colorHover: vars.white,
-    borderColor: vars.brandBlue,
-  })}
-  margin: 0 10px 10px;
+  ${tw`transition duration-200 ease-out border border-solid border-blue text-blue bg-white py-2 px-6 rounded font-normal text-center align-middle min-w-135 cursor-pointer hover:border-blue hover:bg-blue hover:text-white hover:shadow-md hover:no-underline mt-0 mx-3 mb-3`}
 `;
```
Comment (simboonlong) [inline src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx]: explicit transition property

File: src/components/ErrorBoundary.tsx [hunk 6]
```diff
@@ -96,10 +53,12 @@ export class ErrorBoundary extends Component<{}, ErrorBoundaryState> {
           <div id="error">
             <div className="error container">
               <br />
-              <img src="/static/images/errorpage/error.png" style={{ height: "15vh" }} />
+              <img src="/static/images/errorpage/error.png" alt="" style={{ height: "15vh" }} />
               <h2>Something went wrong!</h2>
```
Comment (simboonlong) [inline src/components/ErrorBoundary.tsx]: add some alt text maybe?

File: src/components/AssetManagementPanel/AssetInformationPanel/index.tsx (hunk 2 above)
Comment (isaackps) [reply src/components/AssetManagementPanel/AssetInformationPanel/index.tsx]: ok sure

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx (hunk 3 above)
Comment (isaackps) [reply src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx:10]: ya true

File: src/tailwind.js [hunk 7]
```diff
@@ -1,86 +1,74 @@
...
-      center: true,
-      padding: theme("spacing.4"),
-    }),
     fontFamily: {
-      "source-sans-pro": ["Source Sans Pro", "sans-serif"],
-      montserrat: ["Montserrat", "sans-serif"],
+      sans: ["Roboto", "sans-serif"],
+      display: ["Roboto", "sans-serif"],
+      body: ["Roboto", "sans-serif"],
     },
```
Comment (simboonlong) [inline src/tailwind.js:20]: ^ this is correct/better i think

we should release a patch at commonui side to NOT include font family at all in the exported tw config (we can still defined the family in the root tw config), then let application end to decide for themselves like how this is done now

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx (hunk 4 above)
Comment (isaackps) [reply src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx]: ok

File: src/tailwind.js [hunk 8]
```diff
@@ -1,86 +1,74 @@
...
-          default: "#000000",
+        pink: {
+          default: "#fbc2eb",
         },
-        offblack: {
-          default: "#212529",
+        greyblue: {
+          200: "#e2e8f0",
+          700: "#a7afb7",
+          900: "#343a40",
```
Comment (simboonlong) [inline src/tailwind.js:70]: im not sure about these values, should we move/consolidate them over to commonui after this pr is merged? (should do the same for creator also)

File: src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx (hunk 5 above)
Comment (isaackps) [reply src/components/CertificateDropZone/Views/SharedViewerStyledComponents.tsx]: ok

File: webpack.config.js [hunk 9]
```diff
@@ -92,4 +92,7 @@ module.exports = {
     },
   },
   bail: true,
+  node: {
+    fs: "empty",
+  },
```
Comment (simboonlong) [inline webpack.config.js]: what's this?

File: src/components/ErrorBoundary.tsx (hunk 6 above)
Comment (isaackps) [reply src/components/ErrorBoundary.tsx]: ok

File: src/tailwind.js [hunk 10]
```diff
@@ -1,86 +1,74 @@
...
+const commonUiConfig = require("@govtechsg/tradetrust-ui-components/build/tailwind"); //eslint-disable-line @typescript-eslint/no-var-requires
+
+// https://tailwindcss.com/docs/theme
+const localConfig = {
   // future: {
   //   removeDeprecatedGapUtilities: true,
   //   purgeLayersByDefault: true,
   // },
   // purge: ["./src/**/*.ts", "./src/**/*.tsx"], // not to purge, cos common ui might need certain styles (should purge specific layers instead if really need)
   theme: {
```
Comment (simboonlong) [inline src/tailwind.js]: purge the css?

File: src/tailwind.js (hunk 7 above)
Comment (isaackps) [reply src/tailwind.js:20]: ok

File: src/tailwind.js (hunk 8 above)
Comment (isaackps) [reply src/tailwind.js:70]: ya i think should update tw config in common ui

File: webpack.config.js (hunk 9 above)
Comment (isaackps) [reply webpack.config.js]: my bad, should have removed this, previously was adding twin macro and needed this for smt, but dont need it anymore.

**Comment by simboonlong** [comment]
you may want to pass in `NODE_ENV=production` at  `https://github.com/TradeTrust/tradetrust-website/blob/6fae52794556dd7bc3d4ab8d5c7a16f51e826b5d/package.json#L9` to purge the css

overall, i think we should reduce the reliance on twin macro (understand that we are in the midst of shedding .tsx vars/mixins), so for now it's fine

the problem with twin macro defined styles is that it will take precedence over vanilla tailwind classes, which may lead to some specificity issues later on, we should rely on vanilla tailwind classes as and when possible

File: src/tailwind.js (hunk 10 above)
Comment (isaackps) [reply src/tailwind.js]: oh ya, ok i'll add what u did for creator into here

File: src/components/ErrorBoundary.tsx (hunk 6 above)
Comment (simboonlong) [reply src/components/ErrorBoundary.tsx]: um alt text should be readable text, things like "Logo of Bank of China", "Image of guy standing on stage" etc iirc

File: src/tailwind.js [hunk 11]
```diff
@@ -7,7 +7,9 @@ const localConfig = {
   //   removeDeprecatedGapUtilities: true,
   //   purgeLayersByDefault: true,
   // },
```
Comment (simboonlong) [inline src/tailwind.js]: these commented out can rm too

Comment (isaackps) [reply src/tailwind.js]: ok

**Comment by simboonlong** [comment]
need to use `^1.6.1` at https://github.com/TradeTrust/tradetrust-website/blob/8c3870c32e0bc7b0ca2469eccfee74cfdf640bed/package.json#L44

this will have the `src` folder so the purge can retain the classes that are used in commonui

also i notice some discrepancies after purging, the divider looks weird, should do a quick check in storybook of all the components too

Screenshot 2020-12-17 at 3 51 56 PM

File: package.json [hunk 12]
```diff
@@ -116,6 +117,7 @@
     "@typescript-eslint/parser": "2.3.2",
     "babel-jest": "^25.2.3",
     "babel-loader": "8.0.6",
+    "babel-plugin-macros": "^2.8.0",
     "brotli-webpack-plugin": "^1.1.0",
```
Comment (simboonlong) [inline package.json:121]: wait, double checking wat's this for? i saw `macros` being used as a babel plugin further down, is that how it's being used?

Comment (isaackps) [reply package.json:121]: ya cos without this babel-plugin-macros, all macros will not work

**Review by simboonlong — APPROVED** [review]
lgtm

**Pull request merged**